src/moin/converter/_tests/test_markdown_in.py

```diff
@@ -39,7 +39,7 @@ def setup_class(self):
         (u'<http://moinmo.in/>',
          '<p><a xlink:href="http://moinmo.in/">http://moinmo.in/</a></p>'),
         (u'[yo](javascript:alert("xss"))',
-         '<p>javascript:alert("xss")</p>'),
+         '<p>javascript:alert()</p>'),
```

Comment:
where did the "xss" go?

Reply:
it now comes across as a title attribute and is ignored. That is a rather strange test, old output was malformed, new output is also malformed